src/App/authUtils.ts

```diff
@@ -27,7 +29,22 @@ async function setUser(view: AppTemplate): Promise<string> {
       dispatch({ type: 'SET_USER', data: user.body });
       //   const newToken = jwt.encode(decoded, process.env.HashString || /* istanbul ignore next */'');
       // dispatch({ type: 'GOT_TOKEN', data: { token: newToken, email: auth.email } });
-    } catch (e) { return `${(e as Error).message}`; }
+    } catch (e) {
```

Comment:
please remove this try /catch here, there is no error to catch, just do the dispatch on line 29 inside of the else block, and remove the notification dialog (not needed)